### PR TITLE
Deal with errors while retrieving file from Pagure

### PR DESCRIPTION
**Pull request by majamassarini**
Do not try to parse a Pagure server error response when retrieving packit config file.

Fixes packit/packit-service#1629

### CODE REVIEW

**Review by jpopelka — APPROVED** [review]

File: ogr/services/pagure/project.py [hunk 1]
```diff
@@ -405,6 +405,10 @@ def get_file_content(self, path: str, ref=None) -> str:
 
         if not result or result.reason == "NOT FOUND":
             raise FileNotFoundError(f"File '{path}' on {ref} not found")
+        if result.reason != "OK":
+            raise PagureAPIException(
+                f"File '{path}' on {ref} not found due to {result.reason}"
+            )
```
Comment (lachmanfrantisek) [inline ogr/services/pagure/project.py:411]: Just to check, are we sure that the received `500` errors have the `reason != "OK"`?

**Review by lachmanfrantisek — APPROVED** [review]
Thanks! And well done with the tests..;)

**Review by lbarcziova — APPROVED** [review]

File: ogr/services/pagure/project.py (hunk 1 above)
Comment (lbarcziova) [reply ogr/services/pagure/project.py:411]: Sentry shows `reason: INTERNAL SERVER ERROR`

Comment (lachmanfrantisek) [reply ogr/services/pagure/project.py:411]: Thanks for checking!

**Pull request merged**